src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/ViolationTicketMapper.java

```diff
@@ -0,0 +1,197 @@
+import org.mapstruct.factory.Mappers;
+
+import ca.bc.gov.open.jag.tco.oracledataapi.api.model.ViolationTicket;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.Dispute;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.DisputeCount;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.DisputeStatus;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.ViolationTicketCount;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.YesNo;
+import ca.bc.gov.open.jag.tco.oracledataapi.repository.impl.ords.DisputeRepositoryImpl;
+
+@Mapper
+public interface ViolationTicketMapper {
+	
+	ViolationTicketMapper INSTANCE = Mappers.getMapper(ViolationTicketMapper.class);
+	
+	// Map from Oracle Data API dispute model to ORDS dispute data
+	@Mapping(target = "dispute.entDtm", source = "createdTs", dateFormat = DisputeRepositoryImpl.DATE_TIME_FORMAT)
+	@Mapping(target = "dispute.entUserId", source = "createdBy")
+	@Mapping(target = "dispute.updDtm", source = "modifiedTs", dateFormat = DisputeRepositoryImpl.DATE_TIME_FORMAT)
+	@Mapping(target = "dispute.updUserId", source = "modifiedBy")
+	@Mapping(target = "dispute.disputeId", source = "disputeId")
+	@Mapping(target = "dispute.disputeStatusTypeCd", source = "status", qualifiedByName="mapDisputeStatus")
+	@Mapping(target = "dispute.courtLocationTxt", source = "courtLocation")
+	@Mapping(target = "dispute.issuedDt", source = "issuedDate")
+	@Mapping(target = "dispute.submittedDt", source = "submittedDate")
+	@Mapping(target = "dispute.disputantSurnameNm", source = "disputantSurname")
+	@Mapping(target = "dispute.disputantGiven1Nm", source = "disputantGivenName1")
+	@Mapping(target = "dispute.disputantGiven2Nm", source = "disputantGivenName2")
+	@Mapping(target = "dispute.disputantGiven3Nm", source = "disputantGivenName3")
+	@Mapping(target = "dispute.disputantBirthDt", source = "disputantBirthdate")
+	@Mapping(target = "dispute.disputantOrganizationNm", source = "disputantOrganization")
+	@Mapping(target = "dispute.disputantDrvLicNumberTxt", source = "driversLicenceNumber")
+	@Mapping(target = "dispute.drvLicIssuedIntlProvTxt", source = "driversLicenceProvince")
+	@Mapping(target = "dispute.addressLine1Txt", source = "addressLine1")
+	@Mapping(target = "dispute.addressLine2Txt", source = "addressLine2")
+	@Mapping(target = "dispute.addressLine3Txt", source = "addressLine3")
+	@Mapping(target = "dispute.addressIntlCityTxt", source = "addressCity")
+	@Mapping(target = "dispute.addressIntlProvTxt", source = "addressProvince")
+	@Mapping(target = "dispute.postalCodeTxt", source = "postalCode")
+	@Mapping(target = "dispute.homePhoneNumberTxt", source = "homePhoneNumber")
+	@Mapping(target = "dispute.workPhoneNumberTxt", source = "workPhoneNumber")
+	@Mapping(target = "dispute.emailAddressTxt", source = "emailAddress")
+	@Mapping(target = "dispute.emailVerificationGuid", source = "emailVerificationToken")
+	@Mapping(target = "dispute.emailVerifiedYn", source = "emailAddressVerified", qualifiedByName="mapBooleanToYn")
```

Comment:
This will be a merge conflict with Phil's PR. Let's integrate Phil's first then resolve this one.